specs/PermissionManagement.md

```diff
@@ -0,0 +1,561 @@
+        CHECK_FAILURE(m_webViewProfile6->SetPermissionState(
+            dialog.kind, dialog.origin.c_str(), dialog.state));
+    }
+}
+```
+# API Details
+
+```
+[v1_enum]
+typedef enum COREWEBVIEW2_PERMISSION_KIND {
+
+  // Other permission kinds not shown.
+
+  /// Indicates permission to automatically download multiple files. Permission
+  /// is requested when multiple downloads are triggered at the same time.
+  COREWEBVIEW2_PERMISSION_KIND_AUTOMATIC_DOWNLOADS,
+
+  /// Indicates permission to edit files or folders on the device. Permission
+  /// is requested when developers use the [File System Access API](https://developer.mozilla.org/en-US/docs/Web/API/File_System_Access_API)
+  /// to show the file or folder picker to the end user, and then request "readwrite"
+  /// permission for the user's selection.
+  COREWEBVIEW2_PERMISSION_KIND_FILE_EDITING,
+
+  /// Indicates permission to play audio and video automatically on sites. This
+  /// permission affects the autoplay attribute and play method of the audio and video
+  /// HTML elements, and the start method of the Web Audio API. See the
+  /// [Autoplay guide for media and Web Audio APIs](https://developer.mozilla.org/en-US/docs/Web/Media/Autoplay_guide)
+  /// for details.
+  COREWEBVIEW2_PERMISSION_KIND_AUTOPLAY,
+
+  /// Indicates permission to use fonts on the device. Permission is requested when
+  /// developers use the [Local Font Access API](https://wicg.github.io/local-font-access/)
+  /// to query the system fonts available for styling web content.
+  COREWEBVIEW2_PERMISSION_KIND_LOCAL_FONTS,
+} COREWEBVIEW2_PERMISSION_KIND;
+
+/// This is a continuation of the `ICoreWebView2PermissionRequestedEventArgs`
+/// interface.
+[uuid(08595a19-44f0-41b1-9ae4-5889f5edadcb), object, pointer_default(unique)]
+interface ICoreWebView2PermissionRequestedEventArgs3:
+    ICoreWebView2PermissionRequestedEventArgs2 {
+  /// The permission state set from the `PermissionRequested` event is persisted
+  /// across sessions by default and becomes the new default behavior for future
+  /// `PermissionRequested` events. Browser heurisitics can affect whether the
+  /// event continues to be raised when the state is persisted. Set the
+  /// `ShouldPersist` property to `FALSE` to not persist the state beyond the
+  /// current request, and to continue to receive `PermissionRequested` events
+  /// for this origin and permission kind.
+  [propget] HRESULT ShouldPersist([out, retval] BOOL* value);
+
+  /// Sets the `ShouldPersist` property.
+  [propput] HRESULT ShouldPersist([in] BOOL value);
+}
+
+/// This is the ICoreWebView2 interface for the permission management APIs.
+[uuid(5bdfc5dd-C07a-41b7-bcf2-94020975f185), object, pointer_default(unique)]
+interface ICoreWebView2Profile6 : ICoreWebView2Profile5 {
+  /// Sets permission state for the given permission kind and origin
+  /// asynchronously. The change persists across sessions until it is changed.
```

Comment:
Could you elaborate slightly on 'until it is changed'? Changed by I guess, additional calls to SetPermissionState or the PermissionRequested event? Other ways?